UnitedIncome/components
# Exports DSM Colors

### Description
> Please provide a brief description of your changes below.

Made the colors in the component library up-to-date with all of the shades and tints. I've also exported the `colors.js` file and added an export for `colors.scss` so other projects can have access to the Sass variables.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Instructions will be in the client pull request.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
